Documentation/Books/Manual/Views/ArangoSearch/DetailedOverview.md

```diff
@@ -119,6 +119,33 @@ of removing unused segments after release of internal resources.
   > in-progress ArangoDB transactions will still continue to return a
   > repeatable-read state.
 
+  > ArangoSearch performs operations in its index based on numerous writer
+  > objects that mapped to processed segments. In order to control memory that
+  > is used by these writers (in terms of "writers pool") one can use
+  > `writebuffer*` properties of a view.
+
+- **writebufferIdle** (_optional_; type: `integer`; default: `64`;
+  to disable use: `0`)
+
+  Maximum number of writers (segments) cached in the pool. `0` value turns off
+  caching.
+
+- **writebufferActive** (_optional_; type: `integer`; default: `0`;
+  to disable use: `0`)
+
+  Maximum number of concurrent active writers (segments) performs (a transaction).
+  Other writers (segments) are wait till current active writers (segments) finish.
+  `0` value turns off this limit and used by default. 
+
+- **writebufferSizeMax** (_optional_; type: `integer`; default: `33554432`;
+  to disable use: `0`)
+
+  Maximum memory byte size per writer (segment) before a writer (segment) flush is
+  triggered. `0` value turns off this limit fon any writer (buffer) and will be
+  flushed only after a period defined for special thread during ArangoDB server
```

Comment:
What is "special thread"? Is this an implementation detail the user needs to know at all?
Might be missing an "is":
```suggestion
  flushed only after a period defined for a special thread during ArangoDB server
```